translation.json

```
@@ -0,0 +1,39 @@
+{
+    "widgets": {
+        "emailVerification": {
+            "dataSource": {
+                "offline": "Your device is offline",
+                "email": {
+                    "title": "Verification",
+                    "instruction": "Verify your email address below to gain access.",
+                    "placeholder": "Enter your email",
+                    "progress": "Verifying...",
+                    "actions": {
+                        "send": "Continue",
+                        "userHasCode": "I have a verification code"
+                    }
+                },
+                "phone": {
+                    "instruction": "Enter the verification code that was sent to the phone number associated with"
```

This should include the whole tag content because the `.` punctuation will be translated differently in different languages.

```suggestion
                    "instruction": "Enter the verification code that was sent to the phone number associated with <span class="input-email">\{{ storedEmail }}</span>.",
```